src/ui-shell/header/header-action.component.ts

```diff
@@ -60,5 +60,6 @@ export class HeaderAction extends BaseIconButton {
 		this.active = !this.active;
 		this.selected.emit(this.active);
 		this.activeChange.emit(this.active);
+		this.click.emit(this.active);
```

Comment:
This is redundant. There is a `selected` and an `activeChange` event emitted with the same data. Instead listening for click event, just listen for `selected` or `activeChange` event. You can even double bind a variable with `[active]`.

Reply:
Then is that not an upgrade action that should be documented?
The (click) property on header-action-component seems like it is unused now, and code in the field binding to click breaks when moving from v4 to v5.

Reply:
You're right, we must've missed this. I'll go in and add this to the v4 to v5 migration documentation.

Reply:
Just checked, it never emitted a `click` event. Only the events I mentioned above were emitted. You can verify this here:

You can view the v10 files here: https://github.com/carbon-design-system/carbon-components-angular/blob/v10/src/ui-shell/header/header-action.component.ts